package.json

```diff
@@ -25,21 +26,29 @@
     ]
   },
   "scripts": {
-    "start": "rm -rf dist && yarn build-translations && rollup -c -w",
-    "build": "rm -rf dist && yarn build-translations && rollup -c",
+    "build": "rm -rf dist && yarn build-translations && bob build",
     "docs": "styleguidist build",
-    "test": "jest",
+    "test": "TZ=UTC jest",
```

Comment:
lol

Reply:
haha 

this is how we did it in react sdk - https://github.com/GetStream/stream-chat-react/blob/master/jest-global-setup.js

Reply:
I saw that solution as well, then someone in the thread noted if you are using yarn scripts it's cleaner to just add it to the command.